p2p/peer.py

```diff
@@ -342,7 +343,81 @@ def is_closing(self) -> bool:
     async def _cleanup(self) -> None:
         self.close()
 
+    async def boot(self) -> None:
+        try:
+            await self._boot()
+        except OperationCancelled:
+            # If a cancellation happens during boot we suppress it here and
+            # simply exit without setting the `booted` event.
+            return
+        else:
+            self.booted.set()
+
+    async def _boot(self) -> None:
+        try:
+            await self.ensure_same_side_on_dao_fork()
+        except DAOForkCheckFailure as err:
+            self.logger.debug("DAO fork check with %s failed: %s", self, err)
```

Comment:
Not for logging statements.  We *alway* do `%` style formatting for logging statements because:

1. The logging module does the formatting for us **and** there are APIs to hook into this formatting to do extra *things* (very minor reason)
2. When you hook up logging to something like rollbar or other logging aggregation services, they can only group messages if we do formatting this way.

Reply:
Ah didn't know that!